azure-pipelines.yml

```diff
@@ -418,43 +418,48 @@ stages:
               vs2017_latest89:
                 SQ_VERSION: "LATEST_RELEASE[8.9]"
                 SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
-                SONAR_CSHARPPLUGIN_VERSION: "LATEST_RELEASE"
-                SONAR_VBNETPLUGIN_VERSION: "LATEST_RELEASE"
+                SONAR_CSHARPPLUGIN_VERSION: "DEV"
```

Comment:
This should not be `DEV` but a version released with SQ 8.9

Reply:
See #1450